interop-switch/templates/config.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "interop-switch.fullname" . }}
+  labels:
+      app: {{ template "interop-switch.fullname" . }}
+      chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+      release: {{ .Release.Name }}
+      heritage: {{ .Release.Service }}
+data:
+  SWITCH_SERVICE_HOST: {{ .Values.config.SWITCH_SERVICE_HOST }}
```

Comment:
these config vars are not following the same naming conventions as the other charts.

Although I would argue that perhaps this is not necessary since we will not need the interop-switch component in future. What do you think @ksatya77 ?

Reply:
Yeah, probably. Hoping by mid-PI3 this component is not needed